### PR TITLE
[fix/alert]: fixing showing alerts

### PR DESCRIPTION
**Pull request by elvivi90**
changing the position of the alert to be shown.

### CODE REVIEW

File: src/components/layout/header/Header.jsx [hunk 1]
```diff
@@ -30,15 +30,14 @@ const Header = () => {
...
             <div className={'header-profile'}>
+                
                 <CartHeader cartItems={cartItems} />
-                <div className={'header-profile__container'}>
-                    {user ? (
-                        <LoggedUserHeader logoutHandler={logoutHandler} user={user} />
-                    ) : (
-                        <LoginButton loading={loading} />
-                    )}
+                <div className={'header-profile__container'}>
```
Comment (fitu) [inline src/components/layout/header/Header.jsx:36]: Reformat this part

File: src/App.jsx [hunk 2]
```diff
@@ -11,14 +11,16 @@ import AppRouter from './router/AppRouter';
...
     transtion: transitions.SCALE,
+    offset: '30px',
+    transition: transitions.SCALE
 };
 
 const App = () => {
     const { user, loading } = useSelector((state) => state.auth);
     return (
-        <AlertProvider template={AlertTemplate} {...alertOptions}>
+        <AlertProvider template={AlertTemplate}>
```
Comment (fitu) [inline src/App.jsx:23]: You are missing the alertOptions here.
It should be:
`<AlertProvider template={AlertTemplate} {...alertOptions}>`

With the new settings

File: src/components/layout/header/Header.jsx (hunk 1 above)
Comment (fitu) [reply src/components/layout/header/Header.jsx:36]: Just a tip:
If you want to undo the changes before a commit, in a file, you can run:
`git checkout -- path/to/file` -> eg: `git checkout -- src/components/layout/header/Header.jsx`
or, if you want to undo the changes or set the file as it is in another commits/branch, run:
`git checkout <branch-name> -- /path/to/file` -> eg: `git checkout dev -- src/components/layout/header/Header.jsx`

**Comment by fitu** [comment]
Check **ProductInfo.jsx**:

`alert('Item added to cart!');`

Shouldn't this be `alert.success(...)` ?

File: src/components/layout/header/Header.jsx (hunk 1 above)
Comment (elvivi90) [reply src/components/layout/header/Header.jsx:36]: Change it, according prettier.

File: src/App.jsx (hunk 2 above)
Comment (elvivi90) [reply src/App.jsx:23]: alertOptions added.

**Comment by elvivi90** [comment]
> Check **ProductInfo.jsx**:
> 
> `alert('Item added to cart!');`
> 
> Shouldn't this be `alert.success(...)` ?

**Comment by elvivi90** [comment]
changed !

**Pull request closed without merging**